code/zFulpstationCode/cargo_packs_overwrite.dm

```diff
@@ -0,0 +1,143 @@
+	crate_name = "particle accelerator crate"
+	dangerous = TRUE // Added
+
+/datum/supply_pack/security/armory/ballistic
+	name = "Combat Shotguns Crate"
+	desc = "For when the enemy absolutely needs to be replaced with lead. Contains three Aussec-designed Combat Shotguns, and three Shotgun Bandoliers. Requires Armory access to open."
+	cost = 8000
+	contains = list(/obj/item/gun/ballistic/shotgun/automatic/combat,
+					/obj/item/gun/ballistic/shotgun/automatic/combat,
+					/obj/item/gun/ballistic/shotgun/automatic/combat,
+					/obj/item/storage/belt/bandolier,
+					/obj/item/storage/belt/bandolier,
+					/obj/item/storage/belt/bandolier)
+	crate_name = "combat shotguns crate"
+	dangerous = TRUE // Added
+
+/datum/supply_pack/security/armory/energy
+	name = "Energy Guns Crate"
+	desc = "Contains two Energy Guns, capable of firing both nonlethal and lethal blasts of light. Requires Armory access to open."
+	cost = 2500
+	contains = list(/obj/item/gun/energy/e_gun,
+					/obj/item/gun/energy/e_gun)
+	crate_name = "energy gun crate"
+	crate_type = /obj/structure/closet/crate/secure/plasma
+	dangerous = TRUE // Added
+
+/datum/supply_pack/security/armory/russian
+	name = "Russian Surplus Crate"
+	desc = "Hello Comrade, we have the most modern russian military equipment the black market can offer, for the right price of course. Sadly we couldnt remove the lock so it requires Armory access to open."
+	cost = 5000
+	contraband = TRUE
+	contains = list(/obj/item/reagent_containers/food/snacks/rationpack,
+					/obj/item/ammo_box/a762,
+					/obj/item/storage/toolbox/ammo,
+					/obj/item/clothing/suit/armor/vest/russian,
+					/obj/item/clothing/head/helmet/rus_helmet,
+					/obj/item/clothing/shoes/russian,
+					/obj/item/clothing/gloves/tackler/combat,
+					/obj/item/clothing/under/syndicate/rus_army,
+					/obj/item/clothing/under/costume/soviet,
+					/obj/item/clothing/mask/russian_balaclava,
+					/obj/item/clothing/head/helmet/rus_ushanka,
+					/obj/item/clothing/suit/armor/vest/russian_coat,
+					/obj/item/gun/ballistic/rifle/boltaction,
+					/obj/item/gun/ballistic/rifle/boltaction)
+	crate_name = "surplus military crate"
+	dangerous = TRUE // Added
+
+/datum/supply_pack/security/armory/wt550
+	name = "WT-550 Auto Rifle Crate"
+	desc = "Contains two high-powered, semiautomatic rifles chambered in 4.6x30mm. Requires Armory access to open."
+	cost = 3500
+	contains = list(/obj/item/gun/ballistic/automatic/wt550,
+					/obj/item/gun/ballistic/automatic/wt550)
+	crate_name = "auto rifle crate"
+	dangerous = TRUE // Added
+
+//Goodies overwrite to remove lethals from the list
+
+/datum/supply_pack/goody/dumdum38
+	name = ".38 DumDum Speedloader"
+	desc = "Contains one speedloader of .38 DumDum ammunition, good for embedding in soft targets."
+	cost = 350
+	contains = list(/obj/item/ammo_box/c38/dumdum)
+	special = TRUE // Adding the special = TRUE makes it unable to be bought, to prevent crew with sec gear
```

Comment:
only the detective's revolver can actually use .38 ammo, so why not leave these goodie crates in so that the detective actually stands a chance of being able to order special ammo for his gun?